code/game/objects/items/food/cake.dm

```diff
@@ -456,9 +420,10 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 20, /datum/reagent/consumable/nutriment/vitamin = 5)
 	tastes = list("cake" = 4, "butter" = 2, "cream" = 1)
 	foodtypes = GRAIN | DAIRY | SUGAR
+	slice_type = /obj/item/food/cakeslice/brioche
 
 /obj/item/food/cake/brioche/MakeProcessable()
-	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/cakeslice/brioche, 6, 3 SECONDS, table_required = TRUE)
```

Comment:
Good example of why to make a slice_yield variable, then you wouldn't have to repeat yourself here